x-pack/plugins/cases/docs/openapi/components/schemas/payload_create_case.yaml

```diff
@@ -1,5 +1,7 @@
 type: object
 properties:
+  assignees:
```

Comment:
The `assignees` property is optional (as the severity). Is this captured in the`assignees.yaml` file?

Reply:
All properties are assumed to be optional in an object; if there are required properties, we'd need to add a "required" list per https://swagger.io/docs/specification/data-models/data-types/#object

Reply:
I see. Thanks, Lisa! Should we put the title, the description, etc on the "required" list?